src/htm/utils/Random.hpp

```diff
@@ -87,7 +87,7 @@ class Random : public Serializable  {
     ar( CEREAL_NVP(seed_), 
 	CEREAL_NVP(steps_)
     );  
-    gen.seed(static_cast<UInt64>(seed_)); //reseed
+    gen.seed(static_cast<unsigned int>(seed_)); //reseed
```

Comment:
nit: why not keep using our defines? UInt is used everywhere in the code.

Reply:
> [in Random]  why not keep using our defines? UInt is used everywhere in the code.

I guess I should have used UInt32 but UInt64 gave me a warning.